src/ray/gcs/gcs_server/gcs_job_manager.h

```diff
@@ -104,11 +117,6 @@ class GcsJobManager : public rpc::JobInfoHandler {
 
   /// The cached core worker clients which are used to communicate with workers.
   rpc::CoreWorkerClientPool core_worker_clients_;
-
-  void ClearJobInfos(const rpc::JobTableData &job_data);
```

Comment:
google coding style has recommendation for declaration order: https://google.github.io/styleguide/cppguide.html#Declaration_Order